django/contrib/syndication/views.py

```diff
@@ -86,10 +86,17 @@ def _get_dynamic_attr(self, attname, obj, default=None):
                 code = attr.__code__
             except AttributeError:
                 code = attr.__call__.__code__
-            if code.co_argcount == 2:  # one argument is 'self'
-                return attr(obj)
-            else:
-                return attr()
+            if not code.co_argcount:  # attr was decorated
```

Comment:
I'm not sure if it's correct. What about static methods that have no args:
```python
    @staticmethod
    def item_description():
        return "Overridden item description"
```

Reply:
Thank you! I added a validation with `getattr_static`. Let me know if it's OK

Reply:
And also a comment, I'm afraid this code is too specific

Reply:
We should add a test for `@staticmethods` in advance, because it works without this patch.

Reply:
Makes sense! I'll do it!

Reply:
Done!

https://github.com/django/django/pull/15733